qsiprep/interfaces/dwi_merge.py

```diff
@@ -151,6 +163,16 @@ def _run_interface(self, runtime):
         return runtime
 
 
+def combine_metadata(scan_list, metadata_dict):
```

Comment:
Given that you're not actually combining the metadata, maybe rename this to `select_multirun_metadata` or something similar?

Reply:
I was thinking about adding an option in the future where the slice timings could be averaged or the median taken per slice across the combined scans. I was hoping to get some feedback on that though before implementing

Reply:
I'm not sure how much it matters, because slspec.txt doesn't even contain any timings, just groupings, so I think it just needs the groups to be consistent.

But I can't think of circumstances where the slice timings would be substantially different across runs, unless you're also doing something inadvisable like altering TE.

Reply:
In that case I may just stick with using the exact slice times from the first scan. If all the other scans are within 10ms it shouldn't matter too much. @cookpa should I also add a check in the grouping function to make sure TEs are very similar?

Reply:
Sorry I think I should have said TR there. But if that varies, slice timings probably would too.

I don't know if you want to get into vetting all the imaging parameters before merging, or if that should be a user responsibility

Reply:
Would it make sense to raise an error if the slice times are very different? Something like CuBIDS should probably give the different runs different `acq` values in that situation.

Reply:
I'm about half way through this and am thinking it should go in another PR